client/ayon_deadline/plugins/publish/aftereffects/submit_aftereffects_deadline.py

```diff
@@ -35,6 +35,8 @@ class AfterEffectsSubmitDeadline(
     targets = ["local"]
     settings_category = "deadline"
 
+    multiprocess: bool = True
```

Comment:
What's a good default value @kalisp ?

Reply:
I guess this default value matches the default that the env var would've fallen back to if not set... so.. probably True is good?

Reply:
I think we should get rid of `self.multiprocess` altogether. It is not exposed in Settings.

Originally this field was in Openpype configuration, BUT there was an issue with some version of AE where it should be set differently. We didn't have settings per Variants (we do not have now either), but we had `Environment` fields per Variant. Therefore introduction of `MULTIPROCESS` env var.

I don't think we would like to expose this field in Settings (we dont have way how to do it per Variant), so we should get rid of it.

We should also probably remove default `No` in `Applications`, but thats different issue.